src/include/buffer.h

```diff
@@ -304,7 +310,6 @@ namespace buffer CEPH_BUFFER_API {
 
       void advance(int o);
       void seek(unsigned o);
-      bool operator!=(const iterator_impl& rhs) const;
```

Comment:
I think we need this for abi compat since it was present in infernalis - not entirely sure for these operators though

https://techbase.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B